### ISSUE TITLE
Some feedback

### ISSUE THREAD

**Issue by ashleycox** [chigkim/VOCR#8]
I love this, it works flawlessly and is super useful. This is one of the features that has been sadly missing from Voiceover for too long. Thank you for taking the time to develop it.

I would like to see the ability to change the keyboard shortcut. CMD Shift O is the same shortcut as used to open the documents folder in the finder and to perform tasks in many other applications.

Also, I'm not sure if this is possible. But it would be great to have a command to OCR only the currently focused element - an image for example.

**Comment by chigkim** [comment]
I'm glad that you were able to get it to work without too much trouble.

As a quick fix, I changed the shortcuts to slightly more complicated 
ones to avoid possible conflicts.

command+control+shift+o to start OCR

command+control+arrows to navigate

command+control+shift+left/right to navigate between words

The changes are included in the latest release.

I might change in the future so users can change to different shortcuts.

As far as focused image, it's more complicated because VoiceOver can 
read UI elements that are off the screen thus not visible on the screen.

Thanks for your feedback.
